Database/stylesdb.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
```

Comment:
Since this configuration is required for all of your files, you should put it in a single module, and require it once in your application.